queries/vim/textobjects.scm

```diff
@@ -0,0 +1,20 @@
+(comment) @comment.outer
+
+(function_definition (body) @function.inner) @function.outer
+
+(parameters (identifier) @parameter.inner)
+
+((parameters "," @_start . (identifier) @_end)
+ (#make-range! "parameter.outer" @_start @_end))
+
+((parameters . (identifier) @_start . "," @_end)
+ (#make-range! "parameter.outer" @_start @_end))
+
+(if_statement (body) @conditional.inner) @conditional.outer
+(for_loop (body) @loop.inner) @loop.outer
+(while_loop (body) @loop.inner) @loop.outer
+
+(call_expression) @call.outer
```

Comment:
there isn't a node that contains the inner of the call, was writing a query to match from `(` to `)`, but I think I'll just introduce a new node (`arguments`) on the vim grammar like python has

Reply:
Yup sounds good to have.